### PR TITLE
feat(flow): enhancements to 'add activity' panel

### PR DESCRIPTION
**Pull request by fcastill**
- Activities are sorted alphabetically by title, except for subflow option which always appears first
- Once "add activity" panel is open tab focus is trapped inside the panel
- Allow to navigate items with keyboard
- Activity list is marked as a listbox and can be navigated with arrows, hitting enter will select the option
- Escape key will close the panel

### CODE REVIEW

**Comment by apandurangi** [comment]
There is an additional border for the install activity button: 

image

**Comment by apandurangi** [comment]
Feels like the empty state of activities can be improved in styling? 

image

May be follow something from a different scenario:
image

**Comment by fcastill** [comment]
@apandurangi 

I think the empty state should be addressed separately, I didn't touch it in this PR.

I can't see the border you mention, also there should be a dashed border around the "install new contribution" text but I can't see it in your screenshot. Can you please verify you were trying the correct branch? It should look like this:

Screen Shot 2019-06-17 at 4 16 37 PM

File: libs/plugins/flow-client/src/lib/task-add/activity/activity.component.less [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@background-size: 24px;
+
+:host {
+  display: block;
+  background: url('/assets/svg/flogo.flows.detail.diagram.activity.icon.svg') no-repeat
+    16px;
+  background-size: 24px;
```
[the remark this answers is not in the document]
Comment (fcastill) [reply libs/plugins/flow-client/src/lib/task-add/activity/activity.component.less]: whoops! fixed 😬

**Comment by fcastill** [comment]
I will fix the subflow issue but the other one I'm considering as a won't fix, I did notice during implementation but I don't think it is a valid use case since you usually use the mouse or keyboard for navigation but not both. Maybe I should have mentioned it in the description.

**Comment by apandurangi** [comment]
> I don't think it is a valid use case since you usually use the mouse or keyboard for navigation but not both. Maybe I should have mentioned it in the description.

Okay. Agreed. 👍

**Pull request merged**